app/(service)/my-profile/[userId]/edit/_components/profileEditForm.tsx

```diff
@@ -33,15 +38,31 @@ const ADDRESS_LIST = [
   { id: 24, category: '서울시 송파구' },
   { id: 25, category: '서울시 강동구' },
 ];
+interface FieldValues extends PutProfileBody {}
 export default function ProfileEditForm() {
+  const params = useParams();
+  const { userId } = params;
+  const router = useRouter();
   const {
     register,
     handleSubmit,
     control,
     formState: { errors },
   } = useForm<FieldValues>({ mode: 'all' });
+  const handleOnSubmit = async (data: PutProfileBody) => {
+    try {
+      await putUserProfile(userId, data);
+      alert('프로필을 등록하였습니다');
+      router.push(`/my-profile/${userId}`);
+    } catch (err) {
+      if (err instanceof AxiosError) {
+        const errorMessage = err.response?.data.message;
+        alert(errorMessage);
+      }
+    }
+  };
   return (
-    <form onSubmit={handleSubmit((data) => console.log(data))}>
+    <form onSubmit={handleSubmit((data) => handleOnSubmit(data))}>
```

Comment:
P4:   이렇게 바꿔볼 수 있을 것 같습니당

```suggestion
    <form onSubmit={handleSubmit(onSubmit)}>
```